.gitignore

```diff
@@ -9,6 +9,7 @@ lerna-debug.log*
 
 node_modules
 dist
+site
```

Comment:
Can you add this to the .npmignore too? Thank you!

Reply:
it's  ok